srcpkgs/lsp-plugins/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'lsp-plugins'
+pkgname=lsp-plugins
+version=1.1.30
+revision=1
+build_style=gnu-makefile
+hostmakedepends="pkg-config php"
+makedepends="libsndfile-devel libX11-devel libglvnd-devel lv2 cairo-devel ladspa-sdk
+ jack-devel libstdc++-devel"
+short_desc="Collection of free plugins compatible with LADSPA, LV2 and LinuxVST"
+maintainer="Paper <paper@tilde.institute>"
+license="LGPL-3.0-or-later"
+homepage="https://lsp-plug.in/"
+distfiles="https://github.com/sadko4u/lsp-plugins/archive/${version}/${version}.tar.gz"
+checksum=9cf43257729093c240375b3640b1514dff34b092b83b54a5ee68d7e8565c8f80
+
```

Comment:
Please patch their Makefile, remove all change to `CFLAGS` `CXXFLAGS` for `test` target.